coderd/database/db2sdk/db2sdk.go

```diff
@@ -518,6 +518,7 @@ func Apps(dbApps []database.WorkspaceApp, agent database.WorkspaceAgent, ownerNa
 			},
 			Health: codersdk.WorkspaceAppHealth(dbApp.Health),
 			Hidden: dbApp.Hidden,
+			OpenIn: codersdk.WorkspaceAppOpenIn(dbApp.OpenIn),
```

Comment:
This piece is the most important of the PR - basically extracting data from the db to make them be visible in the SDK and API calls.